lost/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
   testCompile 'org.robolectric:robolectric:3.1.2'
   testCompile 'com.squareup:fest-android:1.0.7'
   testCompile 'org.mockito:mockito-core:1.9.5'
+  compile 'com.android.support:support-annotations:23.4.0'
```

Comment:
We should manually rebase these changes onto master before merging otherwise the support-annotations library will be included twice when the auto-merge resolves.

https://github.com/mapzen/LOST/blob/master/lost/build.gradle#L67